internal/core/data/valuedescriptor.go

```diff
@@ -194,14 +207,9 @@ func decodeValueDescriptor(reader io.ReadCloser) (vd contract.ValueDescriptor, e
 	}
 
 	// Check the formatting
-	match, err := validateFormatString(v)
+	err = validateFormatString(v)
 	if err != nil {
-		LoggingClient.Error("Error checking for format string for value descriptor " + v.Name)
-		return contract.ValueDescriptor{}, err
-	}
-	if !match {
-		LoggingClient.Error("Error posting value descriptor. Format is not a valid printf format.")
-		return contract.ValueDescriptor{}, errors.NewErrValueDescriptorInvalid(v.Name, err)
+		return contract.ValueDescriptor{}, nil
```

Comment:
There's a bug here. Notice that the error is being dropped because instead of returning `err` we're returning `nil`.